projects/notebooks/roles/notebooks_dashboard_scale_test/files/entrypoint.sh

```diff
@@ -91,5 +95,6 @@ echo "$test_exit_code" > "${ARTIFACT_DIR}/test.exit_code"
 
 echo "Test finished with $test_exit_code errors."
 echo "test_execution: $(date)" >> "${ARTIFACT_DIR}/progress_ts.yaml"
+rm ${ARTIFACT_DIR}/Standard Data Science Pipeline-*
```

Comment:
I'm not sure about this one. The spaces won't work, and it's directories

I think this should be in the ansible role, not in this script